ballista/core/src/execution_plans/shuffle_reader.rs

```diff
@@ -547,7 +553,11 @@ fn send_fetch_partitions(
     join_handles.push(tokio::spawn(async move {
         for p in remote_locations.iter() {
             let now = Instant::now();
-            let permit = response_sender.reserve().await.unwrap();
+            let Ok(permit) = response_sender.reserve().await else {
```

Comment:
The same should be added for `send_fetch_partitions`

Reply:
it is I thought?